src/gui/owncloudgui.cpp

```diff
@@ -164,7 +167,7 @@ void ownCloudGui::slotOpenMainDialog()
 
 void ownCloudGui::slotTrayClicked(QSystemTrayIcon::ActivationReason reason)
 {
-    if (reason == QSystemTrayIcon::Trigger || reason == QSystemTrayIcon::Context) {
+    if (reason == QSystemTrayIcon::Trigger) {
```

Comment:
Sorry, but then don't we loose the right click on Mac now?

Really sorry to be annoying like that but since I can't test on that platform...

Reply:
No we don't loose the right click on mac. Just tried it.

Reply:
Woa... not that is an interesting surprise... so I did that change just for Windows in the end. :facepalm: